droid-results/src/main/java/uk/gov/nationalarchives/droid/profile/CsvItemWriter.java

```diff
@@ -107,6 +107,7 @@ public class CsvItemWriter implements ItemWriter<ProfileResourceNode> {
     private static final int MIME_TYPE_ARRAY_INDEX          = 15;
     private static final int FORMAT_NAME_ARRAY_INDEX        = 16;
     private static final int FORMAT_VERSION_ARRAY_INDEX     = 17;
+    private static final String BLANK_SPACE_DELIMITER = " ";
```

Comment:
Do you think you could tweak the formatting of this constant to match the rest of the block of definitions, i.e. align the equals sign with the others.

Reply:
done and pushed the change